<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MensajeOpciones</class>
 <widget class="QWidget" name="MensajeOpciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>295</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>467</width>
    <height>295</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>467</width>
    <height>295</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Soluciones</string>
   </property>
   <widget class="QRadioButton" name="radioButtonOpcion1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>21</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcion 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonOpcion2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcion 2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonOpcion3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcion 3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonOpcion4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Opcion 4</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-4</y>
     <width>21</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Error de conexion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAceptar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
